src/Http/Server.php

```diff
@@ -141,6 +141,22 @@ public function emit(ResponseInterface $response, ?ResponseEmitter $emitter = nu
         $emitter->emit($response);
     }
 
+    /**
+     * Trigger the Server.terminate event.
+     *
+     * The event is used to do potentially heavy tasks after the response is sent
+     * to the client. Only the PHP FPM server API is able to send a response to
+     * the client while the server's PHP process still performs some tasks.
```

Comment:
Should the docs also explain that in non fastcgi environments the terminate event will be triggered before the response is flushed to the client?